operator/internal/manifests/internal/config/options.go

```diff
@@ -32,6 +32,17 @@ type Options struct {
 	ObjectStorage storage.Options
 
 	Retention RetentionOptions
+
+	Overrides map[string]LokiOverrides
+}
+
+type LokiOverrides struct {
+	Stack lokiv1.LimitsTemplateSpec
```

Comment:
This field deserves a better name like `Limits` as it does not represent the stack.